### ISSUE TITLE
Uncaught (in promise) TypeError: _0x509e7f is undefined

### ISSUE THREAD

**Issue by swichers** [IamWarHead/ddb-game-log#106]
When performing a roll on D&D Beyond the following output is present within the console of Foundry:

```
Uncaught (in promise) TypeError: _0x509e7f is undefined
[Detected 1 package: ddb-game-log]
    getActorByDDBEntityId https://example.com/modules/ddb-game-log/dist/main.js:9
    handleFulfilledRoll https://example.com/modules/ddb-game-log/dist/main.js:9
    onClientRoll https://example.com/modules/ddb-game-log/dist/main.js:9
    onMessage https://example.com/modules/ddb-game-log/dist/main.js:9
    <anonymous> https://example.com/modules/ddb-game-log/dist/main.js:9
```

This is in Chrome and Firefox. Looks like it's getting the roll event but something is breaking along the way.

Does this module not support v9 anymore?

**Comment by swichers** [comment]
Same in v10

```
Uncaught (in promise) TypeError: _0x509e7f is undefined
    getActorByDDBEntityId https://example.com/modules/ddb-game-log/dist/main.js:9
    handleFulfilledRoll https://example.com/modules/ddb-game-log/dist/main.js:9
    onClientRoll https://example.com/modules/ddb-game-log/dist/main.js:9
    onMessage https://example.com/modules/ddb-game-log/dist/main.js:9
    <anonymous> https://example.com/modules/ddb-game-log/dist/main.js:9
    _createSocketListeners https://example.com/modules/ddb-game-log/dist/main.js:9
    connect https://example.com/modules/ddb-game-log/dist/main.js:9
    _0x20a580 https://example.com/modules/ddb-game-log/dist/main.js:9
    #call https://example.com/scripts/foundry.js:688
    callAll https://example.com/scripts/foundry.js:645
    setupGame https://example.com/scripts/foundry.js:7769
    _initializeGameView https://example.com/scripts/foundry.js:8820
    _initializeView https://example.com/scripts/foundry.js:8796
    initialize https://example.com/scripts/foundry.js:7630
    async* https://example.com/scripts/foundry.js:78537
    async* https://example.com/scripts/foundry.js:78519
main.js:9:1044621
    onClientRoll https://example.com/modules/ddb-game-log/dist/main.js:9
    AsyncFunctionThrow self-hosted:672
    (Async: async)
    onMessage https://example.com/modules/ddb-game-log/dist/main.js:9
    <anonymous> https://example.com/modules/ddb-game-log/dist/main.js:9
    (Async: EventListener.handleEvent)
    _createSocketListeners https://example.com/modules/ddb-game-log/dist/main.js:9
    connect https://example.com/modules/ddb-game-log/dist/main.js:9
    _0x20a580 https://example.com/modules/ddb-game-log/dist/main.js:9
    InterpretGeneratorResume self-hosted:1458
    AsyncFunctionNext self-hosted:668
    (Async: async)
    #call https://example.com/scripts/foundry.js:688
    callAll https://example.com/scripts/foundry.js:645
    setupGame https://example.com/scripts/foundry.js:7769
    InterpretGeneratorResume self-hosted:1458
    AsyncFunctionNext self-hosted:668
    (Async: async)
    _initializeGameView https://example.com/scripts/foundry.js:8820
    _initializeView https://example.com/scripts/foundry.js:8796
    initialize https://example.com/scripts/foundry.js:7630
    InterpretGeneratorResume self-hosted:1458
    AsyncFunctionNext self-hosted:668
    (Async: async)
    <anonymous> https://example.com/scripts/foundry.js:78537
    AsyncFunctionNext self-hosted:668
    (Async: async)
    <anonymous> https://example.com/scripts/foundry.js:78519
```

No other modules installed.

**Comment by swichers** [comment]
Issue seems to be resolved in v2.0.0

**Comment by IamWarHead** [comment]
> Issue seems to be resolved in v2.0.0

Thanks for the followup!
